src/List/NonEmpty.elm

```diff
@@ -174,19 +174,149 @@ product =
     List.product << toList
 
 
+{-| Put two lists together.
+
+    append (1, [2, 3]) (4, [5]) --> (1, [2, 3, 4, 5])
+
+-}
+append : NonEmptyList a -> NonEmptyList a -> NonEmptyList a
+append ne1 ne2 =
+    case toList ne2 of
+        [] ->
+            ne1
+
+        _ ->
+            foldr cons ne2 ne1
```

Comment:
this comment still applies https://github.com/turboMaCk/nonempty-list-alias-with-zipper/pull/4#r407196149